### PR TITLE
Add touchpoints to app

### PR DESCRIPTION
**Pull request by emmastephenson**
## Related Issue or Background Info

- Fixes app side of #3210 

## Changes Proposed

- Add Touchpoints link to `index.html`

## Screenshots / Demos

Screen Shot 2022-01-19 at 11 04 50 AM
Screen Shot 2022-01-19 at 11 07 17 AM

(note that the copy on this one has changed to I’m interesting in providing more feedback about SimpleReport (usually via a video call).")
Screen Shot 2022-01-19 at 11 05 02 AM

### Infrastructure
- [ ] **Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!**

### Design
- [x] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [x] Any content changes (including new error messages) have been approved by content team

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Review by Rebecca-LM — DISMISSED** [review]
lgtm!

**Review by zdeveloper — DISMISSED** [review]
LGTM! Didn't realize it was that simple :D

**Review by nickclyde — APPROVED** [review]
Looks good! 
1boaF8

**Review by rin-skylight — COMMENTED** [review]
This is a lengthy PR! 😜 LGTM!

**Review by rin-skylight — DISMISSED** [review]
This is a lengthy PR! 😜 LGTM!

**Comment by emmastephenson** [comment]
I had to make some changes to get our e2es passing, if you could re-review I'd appreciate it!

**Comment by Rebecca-LM** [comment]
@emmastephenson did you need me to look at something? not sure what is different? :)

**Review by nickclyde — APPROVED** [review]

**Comment by emmastephenson** [comment]
> @emmastephenson did you need me to look at something? not sure what is different? :)

@Rebecca-LM Oh whoops, I didn't mean to re-request your review :) Nothing has changed from a content perspective, I just added some additional files to get e2e tests passing so it dismissed all the existing reviews.

File: frontend/cypress/.env.e2e.sample [hunk 1]
```diff
@@ -5,4 +5,4 @@ CYPRESS_OKTA_SECRET=
 
 # Smarty streets secrets
 SMARTY_AUTH_ID=
-SMARTY_AUTH_TOKEN=
+SMARTY_AUTH_TOKEN=
```
Comment (zdeveloper) [inline frontend/cypress/.env.e2e.sample:8]: what changed here?

**Review by zdeveloper — APPROVED** [review]
LGTM! with a question

File: frontend/cypress/.env.e2e.sample (hunk 1 above)
Comment (emmastephenson) [reply frontend/cypress/.env.e2e.sample:8]: It was only a whitespace change, but for some reason Git won't drop the file from this PR 🤷

Comment (rin-skylight) [reply frontend/cypress/.env.e2e.sample:8]: There may be a newline character, a null character, or something else that isn't human-detectable. Git is weird.

**Review by rin-skylight — APPROVED** [review]

**Pull request merged**